### ISSUE TITLE
Proposed new tablefmt "fancy"

### ISSUE THREAD

**Issue by tartley** [astanin/python-tabulate#146]
THANK YOU for an awesome library. I'm proposing a PR I'd like to write for you:

I like the existing tablefmt="fancy_grid". But the horizontal lines between each table row take up a lot of vertical space. How would you feel if I were to submit a PR adding a new table format, like "fancy_grid", but without the horizontal lines? eg:

The existing "fancy_grid":

```
>>> print(tabulate(table, headers, tablefmt="fancy_grid"))
╒════════╤═══════╕
│ item   │   qty │
╞════════╪═══════╡
│ spam   │    42 │
├────────┼───────┤
│ eggs   │   451 │
├────────┼───────┤
│ bacon  │     0 │
╘════════╧═══════╛
```

vs the proposed new format:

```
>>> print(tabulate(table, headers, tablefmt="fancy"))
╒════════╤═══════╕
│ item   │   qty │
╞════════╪═══════╡
│ spam   │    42 │
│ eggs   │   451 │
│ bacon  │     0 │
╘════════╧═══════╛
```

Any opinions about what the new table format should be called? I like "fancy", hence the existing "fancy_grid" format would be just like "fancy", but with extra **grid** lines between each table row.

**Comment by tartley** [comment]
It's possible this has already been added in this PR https://github.com/astanin/python-tabulate/pull/80/commits

**Comment by tartley** [comment]
I plan to re-draft this Issue, so that instead of proposing a new tableformat, it instead proposes to create the missing documentation, etc, for the existing "fancy_outline" format.

**Comment by tartley** [comment]
I notice that trying to use 'fancy_outline' as a drop-in replacement for 'pretty' breaks my column center formatting. I should understand why this is.

**Comment by kdeldycke** [comment]
For the record, the `fancy` table formats proposed in this issue have all been implemented by #151, which has just been merged upstream, so this issue has been marked as resolved.
